orchagent/intfsorch.cpp

```diff
@@ -46,7 +46,7 @@ void IntfsOrch::increaseRouterIntfsRefCount(const string &alias)
     SWSS_LOG_ENTER();
 
     m_syncdIntfses[alias].ref_count++;
-    SWSS_LOG_DEBUG("Router interface %s ref count is increased to %d",
+    SWSS_LOG_INFO("Router interface %s ref count is increased to %d",
```

Comment:
why change to INFO?

Reply:
DEBUG comes with function ENTER and EXIT. It is better to separate this with DEBUG level messages which is too verbose.

Reply:
this is still DEBUG level message, does not feel it should be info level. 

---
In reply to: [219416641](https://github.com/Azure/sonic-swss/pull/626#discussion_r219416641) [](ancestors = 219416641)

Reply:
[https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels](https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels)

---
In reply to: [219422213](https://github.com/Azure/sonic-swss/pull/626#discussion_r219422213) [](ancestors = 219422213,219416641)